<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PsmDlgDoctor</class>
 <widget class="QDialog" name="PsmDlgDoctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>医护人员信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1" columnstretch="0,1" rowminimumheight="1,1,1,1,1" columnminimumwidth="0,1">
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>类别：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>编号：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>科室：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leDoctorID"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leDoctorName"/>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pbDoctorDepart">
       <property name="text">
        <string>[选择科室]</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>电话：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="leDoctorPhone"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cmbDoctorType">
       <item>
        <property name="text">
         <string>医生</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>护士</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leDoctorID</tabstop>
  <tabstop>leDoctorName</tabstop>
  <tabstop>cmbDoctorType</tabstop>
  <tabstop>pbDoctorDepart</tabstop>
  <tabstop>leDoctorPhone</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PsmDlgDoctor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
